index.js

```diff
@@ -66,6 +68,14 @@ module.exports = class Autobase {
 
   // Private Methods
 
+  _inputsValidation () {
+    for (const input of this._inputs) {
+      if (input.valueEncoding) {
```

Comment:
&& valueEncoding !== codecs.binary (from the codecs module)

Reply:
✅ done ^^